gradle/dependencies.gradle

```diff
@@ -35,7 +35,7 @@ ext {
     newrelic         : "3.15.0",
     reactiveStreams  : "1.0.2",
     guice            : "4.1.0",
-    springBoot       : "1.3.0.RELEASE",
+    springBoot       : "2.1.3.RELEASE",
```

Comment:
Wouldn't this be considered a breaking change? it will force Spring Boot 2.x on users of the module in a minor version of Ratpack.

Reply:
Yes, it is. I tried working around that with the java 9 stuff seeing if I could do it in 1.x before and Spring was one of those things I simply couldn't move forward.